submissions/AndriyM/task_js_oop/app.js

```diff
@@ -0,0 +1,126 @@
+const FIELD_CONF = {
+    WIDTH: 400,
+    HEIGHT: 400,
+    CELL_WIDTH: 100,
+    CELL_HEIGHT: 82,
+    WATTER_BLOCK: -5,
+    ZERO_POSITION: 0
+};
+const PLAYER_CONF = {
+    PLAYER_WIDTH: 52,
+    SPRITE: "images/char-boy.png",
+    MOVE_LEFT: "left",
+    MOVE_RIGHT: "right",
+    MOVE_DOWN: "down",
+    MOVE_UP: "up",
+    INITIAL_POSITION: {
+        x: FIELD_CONF.WIDTH / 2,
+        y: FIELD_CONF.HEIGHT
+    }
+};
+
+const ENEMY_CONF = {
+    PADDING_AVATAR_X: 75,
+    PADDING_AVATAR_Y: 50,
+    START_POSITION_X: FIELD_CONF.ZERO_POSITION - FIELD_CONF.CELL_WIDTH,
+    END_POSITION_X: FIELD_CONF.WIDTH + FIELD_CONF.CELL_WIDTH,
+    START_POSITION_Y: [60, 145, 228],
```

Comment:
You are tasked to add three more rows for enemies.
Optimize this so it refers to something human-comprehensible (rows) and pixel calculations are done by a computer.